src/utilcode/pedecoder.cpp

```diff
@@ -1790,7 +1790,96 @@ void PEDecoder::LayoutILOnly(void *base, BOOL allowFullPE) const
 
 #endif // #ifndef DACCESS_COMPILE
 
-#ifndef FEATURE_PAL
+DWORD PEDecoder::ReadResourceDictionary(DWORD rvaOfResourceSection, DWORD rva, LPCWSTR name, BOOL *pIsDictionary) const
```

Comment:
Could you do `s/dictionary/directory`? Directory is how Windows names these.

Reply:
Nit: Isn't `LPCWSTR` a Win16-ism and we should say `PCWSTR` these days? i.e. there is only size pointer, not long/far/near/short.